packages/mobile/src/features/common/styles/constans/DifrentEnum.ts

```diff
@@ -1,5 +1,10 @@
+import { Dimensions } from "react-native";
+
 export enum DifrentSlaider {
     WIDTHLARGE = 359,
     WIDTHNORMAL = 172,
     WIDTHSMALL = 81
-}
\ No newline at end of file
+}
+
+const { width: wWidth} = Dimensions.get('window');
```

Comment:
co to?

Reply:
to pozostałości starej myśli :D zaraz usune bo zapomniałam o tym :)